pkg/handlers/license.go

```diff
@@ -342,6 +343,7 @@ func (h *Handler) UploadNewLicense(w http.ResponseWriter, r *http.Request) {
 		uploadLicenseResponse.Success = true
 		uploadLicenseResponse.Slug = a.Slug
 		uploadLicenseResponse.NeedsRegistry = false
+		uploadLicenseResponse.RegistryIsReadOnly = false
```

Comment:
why always `false`? isn't this still supported in online installations?

Reply:
good catch. it's not used. we get registry info on the next page separately.